src/package.test.ts

```diff
@@ -12,7 +12,14 @@ describe("package", () => {
     });
 
     describe("initApp", () => {
-        [".testplane.conf.ts", ".testplane.conf.js", ".hermione.conf.ts", ".hermione.conf.js"].forEach(configName => {
+        [
+            ".testplane.conf.ts",
```

Comment:
you can get names from `CONFIG_NAMES` constant instead of manually synchronizing them here

Reply:
It is good in source files, but i dont think it is in test files. I want small typo in CONFIG_NAMES to break these tests